rust/kernel/error.rs

```diff
@@ -322,6 +322,7 @@ impl Error {
     ///
     /// It is a bug to pass an out-of-range `errno`. `EINVAL` would
     /// be returned in such a case.
+    #[cold]
```

Comment:
With my impress about `#[cold]` attribute, I remembered that it is still unstable right now(in LLVM issue). So it is better to confirm whether this attribute is stable right now.